tests/plugins/test_add_help.py

```diff
@@ -0,0 +1,191 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from __future__ import unicode_literals
+
+import os
+import subprocess
+import pytest
+
+from atomic_reactor.inner import DockerBuildWorkflow
+from atomic_reactor.plugin import PreBuildPluginsRunner
+from atomic_reactor.plugins.pre_add_help import AddHelpPlugin
+from atomic_reactor.util import ImageName, df_parser
+from tests.constants import MOCK_SOURCE
+from tests.fixtures import docker_tasker
+
+import atomic_reactor
+from tests.test_inner import FakeLogger
+from textwrap import dedent
+from flexmock import flexmock
+
+
+class Y(object):
+    pass
+
+
+class X(object):
+    image_id = "xxx"
+    source = Y()
+    source.dockerfile_path = None
+    source.path = None
+    base_image = ImageName(repo="qwe", tag="asd")
+
+
+def generate_a_file(destpath, contents):
+    with open(destpath, 'w') as f:
+        f.write(dedent(contents))
+
+
+@pytest.mark.parametrize('filename', ['help.md', 'other_file.md'])
+def test_add_help_plugin(tmpdir, docker_tasker, filename):
+    df_content = dedent("""
+        FROM fedora
+        RUN yum install -y python-django
+        CMD blabla""")
+    df = df_parser(str(tmpdir))
+    df.content = df_content
+
+    workflow = DockerBuildWorkflow(MOCK_SOURCE, 'test-image')
+    workflow.builder = X
+    workflow.builder.df_path = df.dockerfile_path
+    workflow.builder.df_dir = str(tmpdir)
+
+    help_markdown_path = os.path.join(workflow.builder.df_dir, filename)
+    generate_a_file(help_markdown_path, "foo")
+    help_man_path = os.path.join(workflow.builder.df_dir, AddHelpPlugin.man_filename)
+    generate_a_file(help_man_path, "bar")
+
+    cmd = [u'go-md2man', u'-in={}'.format(help_markdown_path), u'-out={}'.format(help_man_path)]
```

Comment:
With `unicode_literals` imported, do we need to specify `u` here?

Similarly, in the plugin module itself, should we import `unicode_literals`?